litmus-portal/frontend/src/components/WorkflowCard/styles.ts

```diff
@@ -98,7 +98,7 @@ const useStyles = makeStyles((theme) => ({
   },
 
   provider: {
-    color: 'rgba(0, 0, 0, 0.4)',
+    color: theme.palette.providerTextColor,
```

Comment:
https://github.com/litmuschaos/litmus/pull/2120/files#r496476443

Reply:
similar usage as mention in the above comments.